src/game.js

```diff
@@ -1,7 +1,62 @@
 class Game {
+  constructor() {
+    this.player = new Player(0, 0);
+    this.treasure = new Treasure(2, 3);
+    this.score = 0;
+  }
+
+  draw() {
+    clear();
+    this.drawGrid();
+    this.player.draw();
+    this.treasure.draw();
+    if (this.gettingPoints(this.player, this.treasure)) {
+      noLoop();
```

Comment:
Here, the noLoop() and loop() could be omitted since they occur in the same conditional statement! They could be useful if, for instance, you'd want the game to start again "onClick" of a button. 😉